src/components/RichNote.tsx

```diff
@@ -66,8 +66,8 @@ export default function RichNote({
           <RichImgPreview
             src={mediaLink}
             style={{ marginTop: 10 }}
-            width="100%"
             height={140}
+            width={140}
```

Comment:
**issue**: this compresses non-square image unnaturally.

We should find another solution here! I had a play but couldn't quite get it. Will leave it in your capable hands 🙏 

| my image | image rendered in pament memo |
| --- | --- |
|  <img width="504" alt="Screen Shot 2022-07-11 at 2 03 56 PM" src="https://user-images.githubusercontent.com/94939382/178360030-8a037723-1978-49eb-b5ad-75fad0ca9e36.png"> | <img width="552" alt="Screen Shot 2022-07-12 at 6 46 26 AM" src="https://user-images.githubusercontent.com/94939382/178359797-19d0c370-1229-401f-a6c8-08793a6ab6d6.png"> |